src/image.jl

```diff
@@ -10,14 +10,13 @@ function shape(model::ImageClassifier, X, y)
     end
     return (n_input, n_output, n_channels)
 end
-is_embedding_enabled_type(::ImageClassifier) = false
 
 
 build(model::ImageClassifier, rng, shape) =
     Flux.Chain(build(model.builder, rng, shape...),
                model.finaliser)
 
-fitresult(model::ImageClassifier, chain, y) =
+fitresult(model::ImageClassifier, chain, y, ordinal_mappings=nothing, embedding_matrices=nothing) =
```

Comment:
I think you only need to define the one method here:


```julia
fitresult(model::ImageClassifier, chain, y, ordinal_mappings, embedding_matrices) =
```

or you can write

```julia
fitresult(model::ImageClassifier, chain, y, ::Any, ::Any) =
```

Reply:
Done.